de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/InstantIncomingProjectNegotiation.java

```diff
@@ -58,16 +63,19 @@ public InstantIncomingProjectNegotiation(
             projectNegotiationData, sessionManager, session,
             fileReplacementInProgressObservable, workspace, checksumCache,
             connectionService, transmitter, receiver);
-
-        session.registerQueuingHandler(activityQueuer);
     }
 
     @Override
     protected void transfer(IProgressMonitor monitor,
         Map<String, IProject> projectMapping, List<FileList> missingFiles)
         throws IOException, SarosCancellationException {
 
-        awaitActivityQueueingActivation(monitor);
+        try {
+            stoppedLocalEditor = session.getStopManager().stop(
+                session.getLocalUser(), "Read-only while negotiation!");
+        } catch (InterruptedException e) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either re-interrupt this method or rethrow the "InterruptedException". ([squid:S2142](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2142))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=156&issues=AWTIBaM2xXt_iLiSqxxl&open=AWTIBaM2xXt_iLiSqxxl)

[ISSUE_KEY]: AWTIBaM2xXt_iLiSqxxl